.travis.yml

```diff
@@ -1,6 +1,6 @@
 # This config file for Travis CI utilizes https://github.com/ros-planning/moveit_ci/ package.
 sudo: required
-dist: xenial  # distro used by Travis, moveit_ci uses the docker image's distro
+dist: trusty  # distro used by Travis, moveit_ci uses the docker image's distro
```

Comment:
Why should Travis tun on trusty? Shouldn't we use bionic?

Reply:
Travis only provides two distributions: `trusty` and `xenial` (and the latter was only added relatively recently).

Reply:
I even added an in-line comment for this, above! I guess it's not clear enough.
https://github.com/ros-planning/moveit2/pull/77